pkgs/servers/draupnir/test-build.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p nix coreutils
+
+if [ ! -z "$CLEAN" ]; then
+  nix-env -e draupnir
+  nix-store --gc
+  nix-collect-garbage -d
+fi
+
+nix-build -K -A draupnir
+
+if [ ! -z "$INSTALL" ]; then
+  nix-env -e draupnir
+  nix-env -f . -iA draupnir
+fi
+
+if [ ! -z "$RUN" ]; then
+  draupnir
+fi
+
```

Comment:
Did you mean to commit this?

FWIW, it's easier to quickly iterate verifying changes by running `nix-build ../../.. -A draupnir; ./result/bin/draupnir` (or `nix build .#draupnir; ./result/bin/draupnir` or even `nix run .#draupnir` with the fancy new nix command)

Reply:
Yes this was intentional, even if not useful to nixpkgs maintainers